core/DataAccess/LogQueryBuilder.php

```diff
@@ -102,6 +123,30 @@ private function buildWrappedSelectQuery($select, $from, $where, $groupBy, $orde
                 . "Please use a table prefix.");
         }
 
+        $fieldNames = array();
```

Comment:
fixes a bug when an archiver query selects two times `log_action_XYZ.name`. Previously this resulted in an error since the same column name cannot be requested twice and the `as ` statement would be in this case in the outer query etc. There are tests for it...